### DISCUSSION TITLE
Is oppia android going to participate in GSoC 23' ?

### DISCUSSION THREAD

**Discussion by YesShubham01** [oppia/oppia-android#4853] [Q&A (Other)] [answered]
Is oppia android going to participate in GSoC 23' ?

**Comment by yash10019coder** [comment, marked as answer]
No @YesShubham01 oppia-android won't be participating in gsoc this year but oppia-web will participate.

**Comment by MohitGupta121** [comment]
@YesShubham01 please mark the answered comment as answer if you get your answer. If you have any further query feel free to ask.
